### PR TITLE
feat: add placeholders to search

### PR DESCRIPTION
**Pull request by jonasbrunvoll**
Fixes https://github.com/AtB-AS/kundevendt/issues/15750

### Description
- Added placeholder to the search fields in both the 'Plan travel' - and the 'Find departures' - page. The placeholder descriptions translates to norsk, english and nynorsk. 

Planner:
Screenshot 2023-12-12 at 13 47 48

Find departures (mobile view):
Screenshot 2023-12-12 at 13 48 19

### Acceptance criteria
- The placeholder values are precisely formulated.
- No typos.
- Translated into Norwegian, English and Nynorsk.

### CODE REVIEW

File: src/translations/pages/departures.ts [hunk 1]
```diff
@@ -11,6 +11,11 @@ export const Departures = {
         'Kor vil du reise frå?',
       ),
       from: _('Fra', 'From', 'Frå'),
+      placholder: _(
+        'Søk etter adresse, kai eller holdeplass',
+        'Search on address, pier or buss stop',
```
[the remark this answers is not in the document]
Comment (jonasbrunvoll) [reply src/translations/pages/departures.ts:16]: Ops😬

File: src/translations/pages/assistant.ts [hunk 2]
```diff
@@ -13,6 +13,14 @@ export const Assistant = {
       ),
       from: _('Fra', 'From', 'Frå'),
       to: _('Til', 'To', 'Til'),
+      placholder: {
+        from: _(
+          'Søk fra adresse, kai eller holdeplass',
+          'Search from address, pier or buss stop',
+          'Søk frå adresse, kai eller haldeplass',
+        ),
+        to: _('Sluttdestinasjon', 'Final destination', 'Sluttdestinasjon'),
```
[the remark this answers is not in the document]
Comment (mortennordseth) [reply src/translations/pages/assistant.ts:22]: 😄 The app uses "To: Place or address" it seems 

image

[the remark this answers is not in the document]
Comment (jonasbrunvoll) [reply src/translations/pages/assistant.ts:22]: Hva med?
 
Norsk: "Oppgi ankomstdestinasjon" 
Engelsk: "Enter arrival destination" eller "Provide arrival destination"
Nynorsk: "Oppgi ankomstdestinasjon"

**Comment by mikaelbr** [comment]
Naming of PRs should follow conventional commits naming: https://github.com/AtB-AS/mittatb-app/blob/f1a0c1a733302cf128844ac1c327bdd822e6e991/CONTRIBUTING.md#conventional-commits

Haven't added a contributing document to this repo yet I see, but much of the same things apply

File: src/translations/pages/assistant.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (adriansberg) [reply src/translations/pages/assistant.ts]: I think the placeholder probably should not be of a length that is too long for a narrow mobile.

File: src/translations/pages/assistant.ts [hunk 3]
```diff
@@ -13,6 +13,18 @@ export const Assistant = {
...
+      placholder: {
+        from: _(
+          'adresse, kai eller holdeplass',
+          'address, pier or bus stop',
+          'adresse, kai eller haldeplass',
+        ),
+        to: _(
+          'ankomstdestinasjon',
+          'arrival destination',
+          'ankomstdestinasjon',
```
Comment (mikaelbr) [inline src/translations/pages/assistant.ts:25]: Is this modelled from the app? I think this is an example of lack of clear speech in a way. Why is it "ankomstdestinasjon" is that something that people actually say? Have you ever asked anyone "ja, hva er ankomstdestinasjonen din"? 😅 Maybe "ankomststed" is better? Or just the same as from?

File: src/translations/pages/departures.ts [hunk 4]
```diff
@@ -11,6 +11,11 @@ export const Departures = {
         'Kor vil du reise frå?',
       ),
       from: _('Fra', 'From', 'Frå'),
+      placholder: _(
```
Comment (adriansberg) [inline src/translations/pages/departures.ts:14]: placholder → placeholder

File: src/translations/pages/assistant.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (mortennordseth) [reply src/translations/pages/assistant.ts]: Looks good on mobile: 

image

**Review by mikaelbr — APPROVED** [review]
Verified length of text with Product Owner (PO) today. Looks good. Good job

**Pull request merged**